<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>249</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnPersonal">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#btnPersonal{
border: 1px solid transparent;
background-color: transparent;

}</string>
    </property>
    <property name="text">
     <string>Personal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSemillero">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#btnSemillero{
border: 1px solid transparent;
background-color: transparent;

}</string>
    </property>
    <property name="text">
     <string>Semillero</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnContabilidad">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#btnContabilidad {
border: 1px solid transparent;
background-color: transparent;

}</string>
    </property>
    <property name="text">
     <string>Contabilidad</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PAJA System</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>454</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
